styles.css

```diff
@@ -780,7 +780,7 @@ footer {
   .footer_icon {
     display: flex;
     align-items: center;
-    gap: 25px;
+    gap: 15px;
     justify-content: center;
   }
```

Comment:
- Kindly ignore the highlighted piece of code. I am making this comment here so that you can make changes to the following aspects of the form:

### 1)
- [ ] The design of your `textarea` form input and that of the Figma template are not the same. If you look at the template, the `textarea` input has more height. Kindly fix this. 👍 

#### Hint: To achieve this kindly add this HTML attribute to the form input `row="4"`. 💻 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/207446273-b51675c4-e9a3-4aca-887f-7230c48efe0d.png)

### Figma Template

![image](https://user-images.githubusercontent.com/77106962/207446518-c4115ffc-3365-45b7-b4d7-212656b24180.png)

### 2)

- [ ] I also noticed that the `background-image` close to the Get In Touch button is not properly aligned. On the Figma template, it is aligned to the extreme right, but yours is not positioned like that. Kindly fix this.

### Your Design

![image](https://user-images.githubusercontent.com/77106962/207447648-05998afc-673e-4e7e-a59f-cab60e1e03aa.png)

### Figma Template

![image](https://user-images.githubusercontent.com/77106962/207447868-03ccf33c-92ad-4b09-9b61-98bb97a251da.png)

### 3)

- [ ] If you look at the footer of the Figma template, you would notice the space between the last element and the bottom of the mobile page. Yours does not have the space. Kindly implement the design to be identical to that of the template you chose. 👍 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/207448629-d9975c3e-6e12-461c-8095-201010c7fddd.png)

### Figma Template

![image](https://user-images.githubusercontent.com/77106962/207448751-1518f3f4-d636-4cd4-af59-21105f94d748.png)

Reply:
Thank you it is done